CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This is the Developer Changelog for Matomo platform developers. All changes in o
 
 The Product Changelog at **[matomo.org/changelog](https://matomo.org/changelog)** lets you see more details about any Matomo release, such as the list of new guides and FAQs, security fixes, and links to all closed issues. 
 
+## Matomo 4.12.5
```

Comment:
This would actually be 4.13.0 if this is merged before the next release. Otherwise potentially 4.13.1